scripts/gen_changelog.py

```
@@ -0,0 +1,78 @@
+import re
```

Ruff

`re` imported but unused

Remove unused import: `re`

(F401)